injector/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sailplane/injector",
-  "version": "1.0.2",
+  "version": "2.0.0-beta.3",
```

Comment:
Calling attention to the beta version here just in case it should be something else.

Reply:
Changed to 2.0.0 for release.